frontend/document/document.lua

```diff
@@ -152,7 +152,7 @@ end
 
 -- this might be overridden by a document implementation
 function Document:getNativePageDimensions(pageno)
-    local hash = "pgdim|"..self.file.."|"..pageno
+    local hash = "pgdim|"..self.file.."|"..self.mod_time.."|"..pageno
```

Comment:
What was the point again of adding the modification time to some hashes?

(A few weeks ago, I remember I had some crash when opening a PDF from history related to a line with a mod_time. Tried the same action to reproduce it, and couldn't, so I forgot that crash and didn't report it... :/)

Reply:
Custom cover is always `cover.png`. If we set a new image file with different dimensions as a custom cover, muPDF took cached wrong dimensions.